Fix incorrect version number in banner
Patch author: Ryan Finnie <ryan@finnie.org>

"3.5.0" is currently hard-coded (and incorrect)

--- a/src/digitemp.h
+++ b/src/digitemp.h
@@ -18,7 +18,7 @@
    with this program; if not, write to the Free Software Foundation, Inc.,
    59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
    ----------------------------------------------------------------------- */
-#define BANNER_1     "DigiTemp v3.5.0 Copyright 1996-2007 by Brian C. Lane\n"
+#define BANNER_1     "DigiTemp v3.6.0 Copyright 1996-2007 by Brian C. Lane\n"
 #define BANNER_2     "GNU Public License v2.0 - http://www.digitemp.com\n"
 #define BANNER_3     "Compiled for %s\n\n"
 
